file-updates/release-resources/component.yaml

```diff
@@ -0,0 +1,25 @@
+﻿apiVersion: appstudio.redhat.com/v1alpha1
+kind: Component
+metadata:
+  name: fileupdatestest-component
```

Comment:
should probably include namespace, no?

Reply:
since we are using kubeconfig files, the namespace is defined by them